Mongoose/Project 1/app.js

```diff
@@ -0,0 +1,90 @@
+const { json } = require('express');
+const mongoose = require('mongoose');
+
+
+const connectDb = ()=>{
+        return mongoose.connect("mongodb://localhost:27017/Temp")
+}
+
+const readData = async ()=>{
+    let DB = await connectDb();
```

Comment:
No need to connect to db each time here
It should happen on start of the application and schema creation as well, it should happen only once

Reply:
Updated